### ISSUE TITLE
Wrong limits involving logs

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#3773]
Imported from SourceForge on 2024-07-08 15:03:05
Created by **[kcrisman](https://sourceforge.net/u/kcrisman/)** on 2015-06-05 01:29:40
Original: https://sourceforge.net/p/maxima/bugs/2972

---

According to https://groups.google.com/forum/#!topic/sage-support/5nhGpXPyCwk :

Other examples of wrong limits with logarithm in exponent in Sage:
lim((((27**(log(n,3))))/n**3),n=infinity) returns 0 (Wolfram Alpha and Maple return 1).
lim((((27**(log(n,3)+1)))/n**3),n=infinity) returns 0 (Wolfram Alpha and Maple return 27).
lim(((27**(log(n,3)+1)-1)/26+n-log(n,3)-1)/n**3,n=infinity) returns 0 (Wolfram Alpha and Maple return 27/26).
Using SageMathVersion6.7, ReleaseDate:2015−05−17.

~~~~

(%i3) limit( 27^(log(n)/log(3))/n^3, n, inf);
(%o3)                                  0
(%i4) limit( 27^(log(n)/log(3)+1)/n^3, n, inf);
(%o4)                                  0
(%i5) limit( ((27^(log(n)/log(3)+1)-1)/26+n-log(n)/log(3)-1)/n^3,n,inf);
(%o5)                                  0

~~~~

So hopefully somehow Maxima can be "told" about certain identities about logs, at least in limits.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 15:03:07
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2015-06-06 06:02:28
Original: https://sourceforge.net/p/maxima/bugs/2972/#fc21

---

Dunno what's the cause of the errors, but, for the record, `gruntz` returns the same results as cited for WA & Maple.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 15:03:10
Created by **[dgildea](https://sourceforge.net/u/dgildea/)** on 2015-06-25 16:00:46
Original: https://sourceforge.net/p/maxima/bugs/2972/#aca5

---

- **status**: open --> closed
- **assigned_to**: Dan Gildea

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 15:03:14
Created by **[dgildea](https://sourceforge.net/u/dgildea/)** on 2015-06-25 16:00:46
Original: https://sourceforge.net/p/maxima/bugs/2972/#c095

---

Fixed in commit [461825113f3c66b7cea6b0a14d2e2f79d6da2f0b]

call radcan to handle expressions like 3\*log(3)/log(27).
  Incorrect result was caused by sign(log(27)-3\*log(3)) => neg -- too
  much confidence in result of bfloat in sign code.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 15:03:17
Created by **[kcrisman](https://sourceforge.net/u/kcrisman/)** on 2015-06-25 16:10:23
Original: https://sourceforge.net/p/maxima/bugs/2972/#4140

---

Awesome!  Thank you.  (Though downstream not everyone is enthusiastic about radcan, hopefully nothing surprising will come up.)
